content/components/segmented-control.mdx

```diff
@@ -0,0 +1,426 @@
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624845-49eb0b79-21ee-43d0-b4c8-8e8528359427.png"
+      role="presentation"
+    />
+    <Caption>
+      Replace the segmented control with an action menu or use icons that are easily understood without text labels.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624844-c81c94b4-a004-4c89-8aea-2470c429dd75.png"
+      role="presentation"
+    />
+    <Caption>Wrap the buttons to multiple lines, use an overflow button, or remove buttons</Caption>
+  </Dont>
+</DoDontContainer>
+
+## Accessibility
+
+### Keyboard navigation
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+<div>
+
+A segmented control is treated like a [toolbar](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/toolbar_role), not a [radio group](https://developer.mozilla.org/en-US/docs/web/accessibility/aria/roles/radiogroup_role).
+
+<div>
+
+#### `Tab`
+
+The `Tab` key while focused on the previous focusable element moves focus into the segmented control.
+
+When focus is moved into the segmented control, focus should go to the button for whichever choice is selected.
+
+Pressing `Tab` while one of the segmented control buttons is focused will move focus to the next focusable element outside of the segmented control.
+
+#### `Enter` or `Space`
+
+Pressing `Enter` or `Space` while focused on a deselected button selects that button. This is the default behavior for buttons.
+
+If the focused button is already selected, pressing `Enter` or `Space` should not deselect the button.
+
+#### `Left Arrow` or `Right Arrow`
+
+The `Left Arrow` key moves focus to the previous button. If the first button is focused, pressing `Left Arrow` will not move the focus.
+
+The `Right Arrow` key moves focus to the next button. If the last button is focused, pressing `Right Arrow` will not move the focus.
+
+</div>
+
+</div>
+
+<Box flexShrink={0}>
+  <CustomVideoPlayer
+    width="456"
+    autoPlay
+    loop
+    src="https://user-images.githubusercontent.com/2313998/180625732-901564bf-4b2e-4526-a2df-3848fb98d5b3.mp4"
+  />
+</Box>
+</Box>
+
+### Touch targets
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="center"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+
+<Text as="p" mt="0">Ensure segmented control buttons have a large enough touch target size. The buttons should respond to hovers, clicks, and taps anywhere in the touch target area, even if it isn't directly on the control.</Text>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180624850-cd500a01-518a-4f93-81b9-fb38410a4371.png"
```

Comment:
Touch targets need 44x44 min sizes, both for width and height. Icon-only SegmentedControl buttons need to have a `min-inline-size` of `var(--control-minTarget-auto)`.

Reply:
I think you're right, but this didn't come up in the a11y design review. I'll double-check with @ichelsea that we do in fact need the width to be at least 44px.

Reply:
Ok, here's why this passed a11y design review:

> in WCAG 2.2 AA they’re looking at putting a minimum size of 24px by 24px so it would pass that